src/System.Linq.Parallel/tests/ExchangeTests.cs

```diff
@@ -10,183 +11,180 @@ namespace Test
 {
     public class ExchangeTests
     {
-        [Fact]
-        public static void SimplePartitionMergeWhereScanTest()
+        private static readonly ParallelMergeOptions[] Options = new[] {
+            ParallelMergeOptions.AutoBuffered,
+            ParallelMergeOptions.Default,
+            ParallelMergeOptions.FullyBuffered,
+            ParallelMergeOptions.NotBuffered
+        };
+
+        // For each source, test with different degrees of parallelism to force PLINQ to partition.
```

Comment:
If you made these XML comments instead, then when we eventually switch to using ```nameof(PartitioningData)``` instead of ```"PartitioningData```, hovering over ```PartitioningData``` at the usage site will show the description in IntelliSense.